apps/contrib/templatetags/contrib_tags.py

```diff
@@ -29,3 +30,13 @@ def concat_strings(*args):
     for string in args:
         concat_str += str(string)
     return concat_str
+
+
+@register.assignment_tag
```

Comment:
Also this file is now collecting a lot of non-related templates. In the long run we should split the up. Those new tags would perfectly fit into `templatetags/absolute_url.py`.